test/conformance/parallel/consumer-sidecar.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: cloud-proxy-sidecar
+  namespace: openshift-ptp
+  labels:
+    app.kubernetes.io/name: proxy
+spec:
+  containers:
+  - name: sidecar
+    image: quay.io/redhat-cne/cloud-event-proxy
+    args:
+            - "--metrics-addr=127.0.0.1:9091"
+            - "--store-path=/store"
+            - "--transport-host=consumer-events-subscription-service.openshift-ptp.svc.cluster.local:9043"
+            - "--http-event-publishers=ptp-event-publisher-service-master0.openshift-ptp.svc.cluster.local:9043"
```

Comment:
instead of master0,  you  can use NODE_NAME, it will be replaced by referencing to nodename that pod is running on . Again this is only > 4.13  and master support .this  will not work  for any release below 4.13. I sit possible to add that check and skip this tests

Reply:
we might backpor to 4.12 http support

Reply:
This file is not part of the feature, just a manual helper. Will remove the file